extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -15,22 +15,18 @@
  */
 
 import {toggle} from '../../../src/style';
-
-/** @const */
-const TOOLBAR_TARGET_CLASS = 'i-amphtml-toolbar-target';
-
-/** @const */
-const TOOLBAR_ELEMENT_CLASS = 'i-amphtml-toolbar';
+import {toArray} from '../../../src/types';
 
 export class Toolbar {
   /**
   * @param {!Element} element
   * @param {!Window} win
   * @param {!../../../src/service/vsync-impl.Vsync} vsync
+  * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
   */
-  constructor(element, win, vsync) {
+  constructor(element, win, vsync, ampDoc) {
```

Comment:
To avoid confusion, let's remove `win` from arguments here. In rare cases where access to `Window` is needed, you can simply use `ampdoc.win`.

Also, please change naming of `ampDoc` arg to `ampdoc`. I know it's a bit weird, but consistent with the convention.